v3-docs/v3-migration-docs/index.md

```diff
@@ -1,14 +1,244 @@
 
-## Fibers
 
-## Syntax Highlighting
+## What's the status of version 3.0?
 
-```haskell
-add :: Int -> Int -> Int
-add x y = x + y
-```
\ No newline at end of file
+**Latest version:** `3.0-beta.7` <br/>
+**Node.js version:** `20.9.0 LTS`
+
+
+## How to prepare for version 3.0?
+
+You can follow the guide "[How to migrate to Meteor Async in Meteor 2.x](https://guide.meteor.com/prepare-meteor-3.0)" to help you prepare your application for the new version by starting to use async methods.
+
+## What this guide will cover
+
+This guide will try to cover the topics needed to migrate your application to Meteor v3. We will cover the following topics:
+
+- [Breaking Changes](./breaking-changes/index.md), an overview of the changes that will affect your application.
+  - [Meteor.call x Meteor.callAsync](./breaking-changes/call-x-callAsync.md), why should you change your methods to use `Async` methods.
+  - [Upgrading packages](./breaking-changes/upgrading-packages.md), how to upgrade your packages to the be compatible with Meteor v3.
+
+- [How async functions work and how to use them](./api/async-functions.md), a how-to guide in how to use async functions and helpers for Meteor.
+- [Renamed Functions](./api/renamed-functions.md), a list of functions that were renamed in Meteor v3.
+- [Removed Functions](./api/removed-functions.md), a list of functions that were removed in Meteor v3.
+
+- [React in Meteor v3](./front-end/react.md), how to migrate your react code to Meteor v3
+- [Blaze in Meteor v3](./front-end/blaze.md), how to migrate your blaze code to Meteor v3
+
+## What is Fibers?
+
+Meteor was designed at a time when callback hell was a development issue, so the team decided it at the time
+to use [fibers](https://en.wikipedia.org/wiki/Fiber_(computer_science)) to make building applications much more straightforward with synchronous-looking code.
+The Meteor fibers implementation is based on [node-fibers](https://github.com/laverdet/node-fibers), which is no longer supported as of NodeJS v16.0.0.
+
+The main reason for this migration is to remove the dependency on fibers and make Meteor
+compatible with the latest versions of Node.js.
+
+For more information about fibers, you can check this [talk](https://www.youtube.com/watch?v=bxaOGDqVPKw)
+from Ben Newman and this Stack Overflow [answer](https://stackoverflow.com/a/40865153/6688795).
+
+
+
+## What is the Meteor v3 release schedule?
+
+Our current plan is to release Meteor v3 until Q2 2024. This is subject to change as we progress through the development of Meteor v3.
+
+## What is left until Meteor v3 is released?
+
+We have our GitHub discussion where we give updates regarding the development of Meteor v3. You can check it out [here](https://github.com/meteor/meteor/discussions/12865).
+
+The status is the following:
```

Comment:
I think we should avoid duplicating "What is left for Meteor v3" here and maintain it only in the discussion.